src/controllers/users/users.routes.js

```diff
@@ -0,0 +1,131 @@
+const router = require('express').Router();
+const bcrypt = require('bcryptjs');
+const db = require('./users.models.js');
+const jwt = require('../../utils/jwt.js');
+
+router.post('/login',
+  loginInputs,
+  loginDataType,
+  usernameLowerCase,
+  async (req, res, next) => {
+    console.log(req.body);
+    const { username, password } = req.body;
+    try {
+      const user = await db.validate(username)
+      if (user && bcrypt.compareSync(password, user.password)) {
+        const token = await jwt.sign({ id: user.id, username: user.username })
+        res.status(200).json(token);
+      } else {
+        res.status(401).json({ message: 'Invalid Credentials' });
+      }
+    } catch (err) {
+      console.log(err);
+      res.status(500).json(err);
+    }
+  });
+
+
+router.post('/register',
+  registerInputs,
+  registerDataType,
+  usernameLowerCase,
+  usernameIsUnique,
+  hashPassword,
+  async (req, res) => {
+    const userObj = {
+      username: req.body.username,
+      password: req.body.password,
+      height: req.body.height,
+      jump_height: req.body.jumpHeight
+    }
+    try {
+      const arr = await db.insert(userObj);
+      const id = arr[0];
+      if (id) {
+        const token = await jwt.sign({ id, username: req.body.username });
+        res.status(201).json(token);
```

Comment:
You could make a `responseStatusObj = { success: 200 }` somewhere in this file or import it from it's own file, then you could do `res.status(responseStatusObj.success)`, just makes the code that much easier to read.